src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { StyledSwitch } from '../Switch/Switch.styles'
+import { Nav, Menu, SearchBar, Logon } from './NavBar.styles'
+import styled from 'styled-components'
+
+const Container = styled.div`
+
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+`;
+
+export const NavBar = () => {
+    return (
+        <>
+            <Nav>
+                <Menu />
+                <SearchBar />
+                <Container >
+                    <StyledSwitch />
+                    <Logon />
+                </Container>
+            </Nav>
+        </>
+    )
+}
```

Comment:
Generalmente se hace un salto de línea al final de los archivos, aquí te dejo un link con unas cuantas razones para hacerlo https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.